scripts/buildAlloy.cjs

```diff
@@ -142,46 +154,52 @@ alloyComponents.forEach((component) => {
   );
 });
 
-program.action(async ({ inputFile, outputDir, ...modules }) => {
-  const includedModules = Object.entries(modules).reduce(
-    (acc, [key, value]) => {
-      if (value === true) {
-        acc.push(key);
-      }
-
-      return acc;
-    },
-    [],
-  );
+program.action(
+  async ({ inputFile, outputDir, shouldBuildAlloy, ...modules }) => {
+    if (!shouldBuildAlloy) {
+      console.log("shouldBuildAlloy is false, skipping custom alloy build.");
+      return;
+    }
```

Comment:
Same logic as before, just indented by 1 more "tab" and with a guard clause to skip the alloy build if we have any self-hosted alloy instance